lib/rom/sql/plugin/pagination.rb

```diff
@@ -31,7 +31,9 @@ def total
           end
 
           def total_pages
-            (total / per_page) + 1
+            total_pages = total / per_page
+            total_pages += 1 unless total % per_page == 0
+            total_pages
```

Comment:
That one won't work because we are looking for whole ratios (with no decimal positions). Thus the two expectations in the spec ;)

I thought of this oneliner
`(total/per_page) + (total % per_page == 0 ? 0 : 1)`
But seemed too awkward